SECURITY.md

```diff
@@ -2,10 +2,10 @@
 
 ## Supported Versions
 
-As of January 2022 (and until this document is updated), only the v3.0.0 and v4.0.0 _stable_ releases of Strapi are supported for updates. Any previous versions are currently not supported and users are advised to use them "at their own risk".
+As of May 2022 (and until this document is updated), only the v4.x.x _stable_ releases of Strapi are supported for updates and bug fixes. Any previous versions are currently not supported and users are advised to use them "at their own risk".
 
-- v3 support is limited to Critical/High priority security updates only until September 2022
-- v4 is considered LTS until further notice
+- v3.x.x support is limited to Critical/High severity or security updates only until December 2022
```

Comment:
I would remove the "or" in this sentence I think it would make it more clear we only focus on security issues

Reply:
```suggestion
v3.x.x support is limited to Critical/High severity security updates only until December 2022
```